core_lib/src/structure/camera.cpp

```diff
@@ -19,7 +19,7 @@ GNU General Public License for more details.
 Camera::Camera()
 {
 }
-
+// this is kept, since it is used by 'tests'
```

Comment:
Is this still relevant? If we're not using the constructor at all, we might as well remove that one.